config.toml

```diff
@@ -11,32 +11,25 @@ DefaultContentLanguageInSubdir = true
     weight = 1
     languageName = "English"
     [languages.en.params]
-      introTitle = "Welcome to Afkanerd's Blog"
-      introSubtitle = "Afkanerd is and idea, the idea being We code for the people. The ideology evolved form the believe in human rights, freedom fo speech nd expression."
-      introPhoto = ""
+      introTitle = "Afkanerd Blog"
+      introSubtitle = "Afkanerd is an idea, The idea being We code for the people. This ideology evolves from the beliefieve in human rights, freedom of speech, and expression."
```

Comment:
introSubtitle = "Afkanerd is an idea, The idea being We code for the people. This ideology evolves from the belief in human rights, freedom of speech, and expression."